upf/pfcp.c

```diff
@@ -6569,6 +6582,7 @@ static struct pfcp_ie_def tgpp_specs[] =
     SIMPLE_IE(PFCP_IE_QUOTA_HOLDING_TIME, quota_holding_time, "Quota Holding Time"),
     SIMPLE_IE(PFCP_IE_DROPPED_DL_TRAFFIC_THRESHOLD, dropped_dl_traffic_threshold,
 	      "Dropped DL Traffic Threshold"),
+    SIMPLE_IE(PFCP_IE_QUOTA_VALIDITY_TIME, quota_validity_time, "Quota Validity Time"),
```

Comment:
wrong place, should go after PFCP_IE_ALTERNATIVE_SMF_IP_ADDRESS and before PFCP_IE_UE_IP_ADDRESS_POOL_INFORMATION